Assets/Scripts/GeoJsonParser.cs

```diff
@@ -0,0 +1,263 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using GeoJSON.Net;
+using GeoJSON.Net.CoordinateReferenceSystem;
+using GeoJSON.Net.Feature;
+using Netherlands3D.Coordinates;
+using Newtonsoft.Json;
+using SimpleJSON;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.Twin
+{
+    public class GeoJsonParser
+    {
+        public float maxParseDuration = 0.01f;
```

Comment:
even hypothetisch, wat me opvalt is dat 0.01f minder is dan de gemiddelde 60fps frame tijd. krijg je misschien niet een mooier effect als je de maxduration zou zetten naar 0.0165f. Dit is een pure gok ben gewoon benieuwd. Dit mede omdat je nergens Time.deltaTime (maar die is hier waarschijnlijk ook niet van toepassing) hierin terug ziet. Overigens zou het wel kunnen dat de renderloop hem wel meepakt die is onafhankelijk van de update en gaat meestal sneller

Reply:
Mijn idee was om het bewust minder dan 1/60 te maken, want de overige tijd kan dan door andere scripts worden gebruikt om hun ding te doen. Al werkt dat ook niet helemaal meer zoals de bedoeling is, want je kan meerdere parsers hebben en dan krijg ik in de profiler spikes tot 30fps